lib/content-services/src/lib/content-node-selector/content-node-selector-panel.component.ts

```diff
@@ -278,6 +280,14 @@ export class ContentNodeSelectorPanelComponent implements OnInit, OnDestroy {
         return this._chosenNode;
     }
 
+    getSelectedCountTranslation(): string {
+        return this.translation.instant(`NODE_SELECTOR.SELECTED_COUNT`, { count: this.getSelectedCount() });
```

Comment:
you could do the same in the html and let the translate pipe do the job by doing:

`'NODE_SELECTOR.SELECTED_COUNT' | translate: { count: getSelectedCount() }`

Not much different from what you are doing though, just less number of services injected in the constructor in this way as you don't need to inject the translation service, but up to you to decide which one you want to use

Reply:
Good idea, I will change it